### ISSUE TITLE
[Feature request] Scaffolding controller action from routing definition

### ISSUE THREAD

**Issue by gander** [laravel-idea/plugin#332]
Example routing definition:
```php
    Route::get('/foo', 'FooBarController@foo')->name('foo');
    Route::post('/bar/{id}', 'FooBarController@bar')->name('bar');
```
Scaffolding via _Show Context Actions_ menu action, on _action_ fragment:
```php
<?php

namespace App\Http\Controllers;

class FooBarController extends Controller
{
    public function foo()
    {
    }
    public function bar($id)
    {
    }
}
```

**Comment by adelf** [comment]
Yes. It's a good idea. I'm thinking about it a lot. However, I think I can only create a class with 1 method with "Add Missing Controller" quick fix.
And then... maybe add a new quick fix: "Add Missing Method"...

**Comment by gander** [comment]
Yes, that's what I meant, but I didn't know how to describe it. In case this is the first method, it would create the entire controller. In case the controller already existed, it would add a second method.

**Comment by gander** [comment]
@adelf What status?

**Comment by gander** [comment]
Has anything changed since the last comment?

**Comment by gorbunov** [comment]
This feature (create missing method) was released!
